packages/x-data-grid/src/utils/throttle.ts

```diff
@@ -0,0 +1,27 @@
+export interface Cancelable {
+  clear(): void;
+}
+
+export function throttle<T extends (...args: any[]) => any>(func: T, wait = 166) {
```

Comment:
Is this a more performant version of https://github.com/mui/material-ui/blob/19e7b529df30268ea77dc90432a7ef492d9f015e/packages/mui-utils/src/debounce/debounce.ts#L7?

Reply:
It's a different operation. Debounce is going to delay calling its callback until N ms after the last call to debounce. Throttle will call its callback at most once every N ms, but is guaranteed to call it after those N ms.

Reply:
Yeah, I assumed it's a debounce because of the prop name.